docs/contributing/plugins/README.md

```diff
@@ -106,7 +144,8 @@ aspects of the control surface dynamically.
 This binding is usually made using the `bindMatch` method of the provided
 `DeviceShadow` object. This method returns a `ControlShadow` representing the
 control that was bound to the given callbacks. If multiple bindings are needed,
-`bindMatches` can be used. It returns a list of `ControlShadow` objects.
+`bindMatches` can be used. It returns a list of `ControlShadow` objects. For a 
+list of applicable ControlShadow objects, consult [Control Surfaces](control.surface.md).
```

Comment:
I'm not entirely sure that Control Surfaces are relevant here - they're a little different to control shadows.